### PR TITLE
Add types for prismjs/components/

### PR DESCRIPTION
**Pull request by remcohaszing**
Also minimal types were added for `prismjs/components`, because it’s important one should use `prismjs/components/`
instead.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://prismjs.com/#basic-usage-node
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/prismjs/components.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const core: Record<string, any>;
+export const languages: Record<string, any>;
+export const plugins: Record<string, any>;
+export const themes: Record<string, any>;
```
Comment (RunDevelopment) [inline types/prismjs/components.d.ts:4]: Is this export style correct? 

I don't know how TS translates ESM-style exports to commonjs, but `components.js` exports its data like [this](https://github.com/PrismJS/prism/blob/3786f3965cb433cb16d9a2f467337916c28eb8db/components.js#L2).

Comment (remcohaszing) [reply types/prismjs/components.d.ts:4]: Yes this is correct. The following TypeScript:

```ts
import { core } frm 'prismjs/components';

console.log(core);
```

will be transpiled to:

```js
const prismjs_components_1 = require('prismjs/components');

console.log(prismjs_components_1.core);
```

**Review by RunDevelopment — APPROVED** [review]

**Review by amcasey — APPROVED** [review]
Huh, that's the first time I've seen "foo" and "foo/" have different shapes.

**Comment by remcohaszing** [comment]
Ready to merge

**Pull request merged**